### PR TITLE
Send internal mediawiki requests without icache proxy

### PR DESCRIPTION
**Pull request by rybmat**
@Wikia/iwing

### CODE REVIEW

**Review by mszabo-wikia — APPROVED** [review]

File: k8s/k8s-descriptor-template-verify.yaml [hunk 1]
```diff
@@ -68,8 +68,8 @@ spec:
             value: prod
           - name: WIKIA_DATACENTER
             value: sjc
-          - name: INTERNAL_CACHE_HOST
-            value: prod.icache.service.sjc.consul
+          - name: MW_INTERNAL_HOST
+            value: mediawiki-verify
```
Comment (macbre) [inline k8s/k8s-descriptor-template-verify.yaml:76]: There is no verify on k8s

Comment (rybmat) [reply k8s/k8s-descriptor-template-verify.yaml:76]: @macbre so should we sunset verify env on mobile-wiki?

Comment (macbre) [reply k8s/k8s-descriptor-template-verify.yaml:76]: I believe so

**Comment by mszabo-wikia** [comment]
This was done in https://github.com/Wikia/mobile-wiki/pull/1633

**Pull request closed without merging**